hw1/02-palindrome.js

```diff
@@ -1,3 +1,22 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
+const outputDiv = document.querySelector("div:last-child");
 
-elem.addEventListener('input', handleInput);
+function handleInput() {
+  const input = Number(elem.value);
+
+  if (isNaN(input) || input <= 0) {
+    outputDiv.innerHTML = '';
+    return;
+  }
```

Comment:
It's helpful to show the user a more specific error message so they know what went wrong. Instead of leaving a blank space when someone types in a negative number, you can show a message explaining the problem.